lib/cli/commands/validate.js

```diff
@@ -46,16 +46,21 @@ module.exports = function (opts) {
           argv['base-path'],
           argv.fetch);
         if (supportData.resolved) {
-          const supportDataJSON = JSON.parse(supportData.contents);
           try {
-            const result = support.validate(supportDataJSON, true);
-            if (result) {
-              console.log('Your support information is valid!');
-              return;
+            const supportDataJSON = JSON.parse(supportData.contents);
+            try {
+              const result = support.validate(supportDataJSON, true);
+              if (result) {
+                console.log('Your support information is valid!');
+                return;
+              }
+            } catch (e) {
+              console.log('Error in support JSON');
```

Comment:
console.error instead of console.log?

Reply:
@ljharb this is consistent with the rest of the usage in the file. 

We should evaluate changing that across the board.  I've opened this issue to do that as it may require additional changes to the test harness as well: https://github.com/pkgjs/support/issues/27.

For this PR I think we should just stay consistent and bring everything that needs to go to console.error over in https://github.com/pkgjs/support/issues/27.